Sources/UIKit/StyleableUIElement.swift

```diff
@@ -224,3 +224,31 @@ internal class StringStyleHolder: NSObject {
     }
 
 }
+
+extension Dictionary where Key: RawRepresentable, Key.RawValue == String {
+
+    var withStringKeys: [String: Value] {
+        var newDict: [String: Value] = [:]
+        for (key, value) in self {
+            newDict[key.rawValue] = value
+        }
+
+        return newDict
+    }
+
+}
+
+extension Dictionary where Key == String {
+
+    func withTypedKeys<KeyType>() -> [KeyType: Value] where KeyType: RawRepresentable, KeyType.RawValue == String {
```

Comment:
These are some super handy helpers :)

Reply:
I don't remember writing these! Looks like I did, though, unless I found them somewhere and forgot to give credit. But I'm usually pretty good about that when pasting code.